src/tests/end-to-end/common/browser.ts

```diff
@@ -128,11 +128,16 @@ export class Browser {
             t => t.type() === 'background_page' && Browser.isExtensionBackgroundPage(t.url()),
         );
 
-        this.memoizedBackgroundPage = new Page(await backgroundPageTarget.page());
+        this.memoizedBackgroundPage = new Page(await backgroundPageTarget.page(), { onPageCrash: this.onPageCrash });
 
         return this.memoizedBackgroundPage;
     }
 
+    private onPageCrash = () => {
+        const errorMessage = `!!! Browser.onPageCrashed: see detailed chrome logs '${browserLogPath(this.browserInstanceId)}'`;
+        console.log(errorMessage);
```

Comment:
should we use `console.error` here??

Reply:
Practically speaking, I think Jest outputs them the same way, but I think it's more descriptive so I agree with this idea :)

Reply:
updated